librz/include/rz_vector.h

```diff
@@ -89,6 +89,18 @@ static inline void *rz_vector_index_ptr(RzVector *vec, size_t index) {
 	return (char *)vec->a + vec->elem_size * index;
 }
 
+// returns a pointer to the head of the vector
```

Comment:
Better document what head/tail actually means than just repeating what the function name already says.